### DISCUSSION TITLE
LAB03.01

### DISCUSSION THREAD

**Discussion by JuanDavidSanchezA** [rramosp/ai4eng.v1#155] [03 - GENERACION DE MODELOS]
Hola, no he podido iniciar este laboratorio. Cuando ejecuto la primera celda se queda y  no permite pasar a la celda del log.

Agradezco ayuda con este tema para poder continuar con los módulos

**Comment by Felipe-RA** [comment]
Saludos JuanDavidSanchezA

Necesitamos más información para poderte ayudar (capturas de pantalla con el error o lo que te pasa, etc.)

¿Probaste a reiniciar el kernel?

Estamos pendientes a más información!

**Reply by JuanDavidSanchezA** [reply]
Ahora lo que me pasa es que pasa la primera vez a la celda del log y cuand doy click en el link para autenticar con google me despliega este error
![image](https://user-images.githubusercontent.com/108543458/177423222-480fa8c8-2668-4d34-9a1d-5eca792677f4.png)

**Comment by Felipe-RA** [comment]
En este momento el acceso usando google no funciona.

Por favor usa usuario y contraseña.

**Reply by JuanDavidSanchezA** [reply]
Gracias

Pero yo nunca he podido iniciar sesión por ese método, siempre me ha salido este error

![image](https://user-images.githubusercontent.com/108543458/177427482-e2dcb097-5e54-4828-ae88-736b5c76c096.png)

Por lo que he optado por iniciar sesión con autenticación de google

**Reply by Felipe-RA** [reply]
Como indica tu mensaje de error, el problema esta en tu usuario/contraseña

¿Ya probaste a cambiarla?

**Reply by JuanDavidSanchezA** [reply]
Si, cambié mi contraseña en mi cuenta de google y continúa el mismo error.

Pero pregunto. Este correo y contraseña que solicita para iniciar sesión son las mismas credenciales de goolge? De ser así, no entiendo porqué no me permite el inicio de sesión

**Comment by rramosp** [comment]
no en la cuenta de google, sino en la de la plataforma rlxmooc

On Thu, Jul 7, 2022 at 9:38 AM JuanDavidSanchezA ***@***.***>
wrote:

> Si, cambié mi contraseña en mi cuenta de google y continúa el mismo error.
>
> Pero pregunto. Este correo y contraseña que solicita para iniciar sesión
> son las mismas credenciales de goolge? De ser así, no entiendo porqué no me
> permite el inicio de sesión
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/155#discussioncomment-3100929>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKG64ZPCT2ITQG6EITVS3TX7ANCNFSM52NK7C7A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by JuanDavidSanchezA** [reply]
Hola, muchas gracias por la ayuda, con eso ya pude ingresar

**Comment by rramosp** [comment]
vete a la plataforma y haz un reset de tu password

On Tue, Jul 5, 2022 at 5:29 PM JuanDavidSanchezA ***@***.***>
wrote:

> Gracias
>
> Pero yo nunca he podido iniciar sesión por ese método, siempre me ha
> salido este error
>
> [image: image]
> <https://user-images.githubusercontent.com/108543458/177427482-e2dcb097-5e54-4828-ae88-736b5c76c096.png>
>
> Por lo que he optado por iniciar sesión con autenticación de google
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/155#discussioncomment-3088158>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIMB7CSI4ZAIYG7C7MTVSSZL7ANCNFSM52NK7C7A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
